server.js

```diff
@@ -88,6 +88,13 @@ io.on("connection", client => {
 
   // TODO Create a subscription for this new client to some occupancy changes
   // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
+  // TODO subscribe to realOccupancyChanges AND simulatedOccupancyChanges
+  const sub = simulatedOccupancyChanges.subscribe(notifyClient);
```

Comment:
An alternative to this style is discussed in the [WebSockets Case Study](https://github.com/deanius/rx-helper/wiki/Case-Study:-Web-Sockets. In the alternate style, you must assign the subscription a unique name such as `client-${client.id}` - here we simply use a raw RxJS subscription instead of an Antares/Rx-Helper renderer. In either case, the behavior, and how you unsubscribe is the same. TMTOWTDI.